backend/app/tests/services/test_stats.py

```
@@ -0,0 +1,67 @@
+def test_post_to_discord_noop_when_webhook_unset():
+    with patch.object(stats_mod.settings, "DISCORD_STATS_WEBHOOK_URL", None), patch(
+        "app.services.stats.requests.post"
+    ) as mock_post:
+        post_to_discord(["anything"])
+        mock_post.assert_not_called()
+
+
+def test_post_to_discord_packs_sections_under_size_limit():
+    posted: list[str] = []
+
+    def fake_post(url, json, timeout):
```

Ruff (0.16.0)

[warning] Unused function argument: `url`

(ARG001)